### ISSUE TITLE
MutableMapping is not a 100% replacement for DictMixin

### ISSUE THREAD

**Issue by cclauss** [internetarchive/openlibrary#2889]
https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/upstream/utils.py#L6
https://github.com/infogami/infogami/issues/6
```
______ ERROR collecting openlibrary/plugins/upstream/tests/test_utils.py _______
1672ImportError while importing test module '/home/travis/build/internetarchive/openlibrary/openlibrary/plugins/upstream/tests/test_utils.py'.
1673Hint: make sure your test modules/packages have valid Python names.
1061Traceback:
1062openlibrary/plugins/upstream/tests/test_utils.py:2: in <module>
1063    from .. import utils
1064openlibrary/plugins/upstream/utils.py:6: in <module>
1065    from UserDict import DictMixin
1066E   ModuleNotFoundError: No module named 'UserDict'
```
### Steps to Reproduce

1. Go to ...
2. Do ...

* Actual: 
* Expected: 

### Details

- **Logged in (Y/N)?** 
- **Browser type/version?** 
- **Operating system?** 
- **Environment (prod/dev/local)?** prod

**Comment by tfmorris** [comment]
I'm struggling with the extreme brevity and the apparent mismatch between the issue title and the code. 

Is the solution not the same as what was used for infogami?

```
try:
    from collections.abc import Mapping
except ImportError:
    from collections import Mapping
```

and then use `Mapping` instead of `DictMixin` for the only reference?

**Comment by cclauss** [comment]
You are correct.  `from six.moves.collections_abc import Mapping` seems to work at https://github.com/internetarchive/openlibrary/pull/2888/commits/c0ccae0d6b36a36684edeae283c4ef9070ef6978

Sorry.  I had thought that you had done something more complex to fix infogami/infogami#6
